server/sonar-server/src/main/java/org/sonar/server/organization/ws/RemoveMemberAction.java

```diff
@@ -96,6 +99,8 @@ public void handle(Request request, Response response) throws Exception {
 
       dbClient.organizationMemberDao().delete(dbSession, organizationMember.getOrganizationUuid(), organizationMember.getUserId());
       dbSession.commit();
+
+      userIndexer.index(user.getLogin());
```

Comment:
Permission is not in user index => indeed it's a good point that disabling a user/removing membership will not update permission indexes in issues/project/project measures index.
We could ask Atlas, but I don't we could do anything, permission index will be updated on next analysis.

So you can remove user indexing here.